psp-bugos/Controllers/AccountController.cs

```diff
@@ -0,0 +1,49 @@
+using Microsoft.AspNetCore.Mvc;
+using psp_bugos.Models;
+using psp_bugos.RandomDataGenerator;
+
+namespace psp_bugos.Controllers;
+
+[ApiController]
+public class AccountController : Controller
+{
+    private readonly IRandomDataGenerator m_randomDataGenerator;
```

Comment:
```suggestion
    private readonly IRandomDataGenerator _randomDataGenerator;
```
Here and elsewhere, _camelCase for private fields.

Reply:
My bad, fixed